src/oc_erchef/rebar.config

```diff
@@ -54,7 +54,7 @@
         {jiffy, ".*",
          {git, "https://github.com/joedevivo/jiffy", {branch, "master"}}},
         {ibrowse, ".*",
-         {git, "https://github.com/opscode/ibrowse", {tag, "v4.0.1.1"}}},
+         {git, "https://github.com/cmullaparthi/ibrowse", {branch, "master"}}},
```

Comment:
Do we have to pull in the new version of ibrowse for this to work? Considering how much we all love ibrowse around here, it seems like quite a large undertaking to swallow ~2.5 years worth of change from that project.